src/components/Driver/DriverPage.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { useHistory, useLocation } from 'react-router-dom';
+import Maps from '../../shared/Maps/Maps'
+
+const DriverPage = () => {
+  const history = useHistory()
+  const location = useLocation()
+  
+  const onSelect = (event) => {
```

Comment:
switch statement based on element innerText can cause bugs in future, if the text changes

Reply:
Ok, I changed.